OmniKit/MessageTransport/MessageBlocks/PodInfoFaultEvent.swift

```diff
@@ -61,16 +62,16 @@ public struct PodInfoFaultEvent : PodInfo, Equatable {
         let reservoirValue = Double((Int(encodedData[11] & 0x3) << 8) + Int(encodedData[12])) * podPulseSize
         
         if reservoirValue <= StatusResponse.maximumReservoirReading {
-            self.reservoirLevel = reservoirValue
+            self.reservoirLevel = String(format:"%.2f", reservoirValue)
         } else {
-            self.reservoirLevel = nil
+            self.reservoirLevel = "50+"
         }
         
         self.timeActive = TimeInterval(minutes: Double(encodedData[13...14].toBigEndian(UInt16.self)))
         
-        self.previousStatus = FaultEventCode(rawValue: encodedData[15])
+        self.unacknowledgedAlerts = Data(encodedData[15])
```

Comment:
If this is a single value, then it should be an integer type.

Reply:
fixed